dev-python/libsass/libsass-0.19.4.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A straightforward binding of libsass for Python"
+HOMEPAGE="https://github.com/sass/libsass-python"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="dev-libs/libsass
+		dev-python/six[${PYTHON_USEDEP}]
+		test? ( dev-python/PyQt5[testlib,${PYTHON_USEDEP}] )"
```

Comment:
```
>>> Test phase: dev-python/libsass-0.19.4
 * python3_6: running distutils-r1_run_phase python_test
Traceback (most recent call last):
  File "sasstests.py", line 22, in <module>
    from werkzeug.test import Client
ModuleNotFoundError: No module named 'werkzeug'
```
It also seems to need setuptool in RDEPEND, you can use `DISTUTILS_USE_SETUPTOOLS rdepend` before inheriting *distutils-r1.eclass.*

I'll add these, rest looks good to me. Thanks!